Mixxx Coorporation with AI Research Project (#388)
mixxxdj/website: content/news/XXXX-XX-XX-Mixxx_cooperation_AIResearch.md

title: Demand for cooperation in AI research project
authors: Evelynne Veys
tags: AI, cooperation research project.
comments: yes
status: draft

#### Mixxx has been asked to cooperate in a AI-Research Project 'Observe & Serve the audience'.

> Mixxx has been contacted by a joint research project between a university and a major AI company with the question if Mixxx wants to cooperate by asking DJ's to help them. Mixxx agreed.

Reviewer: ```suggestion
Mixxx has been contacted by a joint research project between a university and a major AI company with the question if Mixxx wants to cooperate by asking DJ's to help them. On benevolent behalf of all our users, Mixxx agreed.
```

The research project wants to improve the detection of human sentiments, more particular individuals located in a crowd.
As most of the project is still under embargo, we can only give a small description:

> - They are looking for +/- 25 DJ's that want to participate to the research project by installing the research gear on their gigd.

Reviewer: gigd?

Author: > gigd?

gigs

> - The DJ's would receive a computer with two heat-detecting camera's, one regular camera and a HQ microphone..

Reviewer: ```suggestion
- The DJ's would receive a computer with two heat-detecting cameras, one regular camera and a HQ microphone.
```

> - The camera's need to be pointed at the public.

Reviewer: An apostrophe denotes "possession", except in the case of "it's" where it means "it is".

```suggestion
- The cameras need to be pointed at the public.
```

Reviewer: "DJ's" or "DJs" is correct for plural since DJ is an acronym

Author: > An apostrophe denotes "possession", except in the case of "it's" where it means "it is".

Automatism, camera's = plural of camera in dutch

> - Observing the audience DJ's would get some on-screen demands to play cheerfull / sad / uptempo or slow music, the camera's will observe emotions.

Reviewer: ```suggestion
- Observing the audience DJ's would get some on-screen demands to play cheerful / sad / uptempo or slow music, the cameras will observe emotions.
```

- At some points the system can even demand to play a complete other genre than the public wants (to detect disappointment and happiness).
- DJ's will have to install the research gear at least five times.
- DJ's will be rewarded for their cooperation. How is not communicated to us.

> - The Public needs to be warned that their emotions can be recorded for research.

Reviewer: maybe something like

```suggestion
- The Public needs to be warned that their emotions can be recorded for research.
- Audience members agree to have their likeness and appearance recorded and analyzed for the purpose of generating updated models.
- Audience members agree to be replicated and replaced as deemed necessary by the project. 
```
